documentation/guide-beanmapping.asciidoc

```diff
@@ -19,57 +19,17 @@ public UserEto mapUser(UserEntity source) {
 ----
 
 Therefore we are using a `BeanMapper` for this purpose that makes our lives a lot easier.
+There are several bean mapping frameworks with different approaches.
 
-== Bean-Mapper Dependency
-To get access to the `BeanMapper` we have to use either of below dependency in our POM:
+For a link:spring.asciidoc[devon4j-spring] application we recommend Orika, follow link:spring/guide-beanmapping-spring.asciidoc[Spring Bean-Mapping] for an introduction to Orika and Dozer in a devon4j-spring context application.
 
-We started with dozer.sourceforge.net/[dozer] in devon4j and still support it. However, we now recommend https://orika-mapper.github.io/orika-docs/[orika] (for new projects) as it is much faster (see https://www.baeldung.com/java-performance-mapping-frameworks#2-orika[Performance of Java Mapping Frameworks]).
+NOTE: devon4j started with http://dozer.sourceforge.net/[Dozer] as framework for Spring applications and still supports it. However, we now recommend Orika (for new projects) as it is much faster (see https://www.baeldung.com/java-performance-mapping-frameworks#2-orika[Performance of Java Mapping Frameworks]).
 
-[source,xml]
-----
-    <dependency>
-      <groupId>com.devonfw.java</groupId>
-      <artifactId>devon4j-beanmapping-orika</artifactId>
-    </dependency>
-----
+For a link:quarkus.asciidoc[Quarkus] application we recommend Mapstruct, follow xref:quarkus/guide-beanmapping-quarkus.asciidoc[Quarkus Bean-Mapping] for an introduction to Mapstruct in a quarkus context application.
```

Comment:
Be consistent when you link to the guide. Use either link or xref, not both.
Quarkus with initial capital letter. Check also the other guides.